benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -906,20 +908,27 @@ void GraphicsBenchmarkApp::ProcessKnobs()
     }
     pAllTexturesTo1x1->SetVisible(enableSpheres && (pKnobPs->GetValue() == SpherePS::SPHERE_PS_MEM_BOUND));
 
-    // Update sphere resources and mesh
-    if (setupSphereResourcesAndMeshes) {
-        SetupSphereResources();
-        SetupSphereMeshes();
-    }
-
-    // Rebuild pipelines
-    if (rebuildSpherePipeline) {
-        SetupSpheresPipelines();
-    }
-
-    // Update descriptors
-    if (updateSphereDescriptors) {
-        UpdateSphereDescriptors();
+    if (enableSpheres) {
+        // Update sphere resources and mesh
+        const bool spheresAreSetUp = mSpheresAreSetUp;
+        if (!spheresAreSetUp) {
```

Comment:
I think we can actually inline now and just check !mSpheresAreSetUp here

Reply:
Done.